.travis.yml

```diff
@@ -1,9 +1,11 @@
 # Travis continuous integration configuration file
 matrix:
   include:
+  - dist: xenial
   - language: python
     python:
       - "2.7"
+      - "3.7"
```

Comment:
This isn't how to include a new build in a build matrix, sorry. You need to copy the python block. (Build matrices have different syntax to normal travis files.)